roles/aws.ec2-autoscaling-group/defaults/main.yml

```diff
@@ -15,6 +15,7 @@ asg_max_size: 1
 asg_desired_capacity: 1
 asg_subnets: []
 asg_launch_config_instance_size: "t2.small"
+asg_launch_config_root_volume_size: 16 # Volume of the root volume on the ecs instance ( in GB )
```

Comment:
could do, but 8 is pretty small and we keep hitting this limit. 16gb just gives a bit more space by default.